yaks/spec/spec_helper.rb

```diff
@@ -20,6 +21,7 @@
 
 
 RSpec.configure do |rspec|
+  rspec.mock_framework = :rspec
```

Comment:
bogus has the same functionality as rspec-mocks, so we should stick to one of them.